pennylane/devices/autograd_ops.py

```diff
@@ -180,3 +180,56 @@ def MultiRZ(theta, n):
         array[complex]: diagonal part of the multi-qubit rotation matrix
     """
     return np.exp(-1j * theta / 2 * pauli_eigs(n))
+
+
+def SingleExcitation(phi):
+    r"""
+    Single excitation rotation.

```

Comment:
This is a recurring suggestion across files.
```suggestion
    r"""Single excitation rotation.
```